src/modules/esl-carousel/renderers/esl-carousel.default.renderer.less

```diff
@@ -8,9 +8,9 @@
   }
 
   &.esl-carousel-horizontal [esl-carousel-slide] {
-    width: var(--esl-slide-size);
+    width: var(--esl-slide-size, --esl-slide-fallback-size, 100%);
```

Comment:
It seems to me that the user should use this variable `--esl-slide-fallback-size` to preset the slide intrinsic size. But the name sounds strange...

Reply:
This size will be used until carousel initialised (no JS) so then it fallback (non primary source)

Reply:
As an idea `--esl-slide-initial-size`

Reply:
I like it better this way.